logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/JoinExistingMLSConversationUseCase.kt

```diff
@@ -87,7 +87,13 @@ internal class JoinExistingMLSConversationUseCaseImpl(
                     if (failure.kaliumException.isMlsStaleMessage()) {
                         kaliumLogger.w("Epoch out of date for conversation ${conversation.id}, re-fetching and re-trying")
                         // Re-fetch current epoch and try again
-                        conversationRepository.fetchConversation(conversation.id).flatMap {
+                        if (conversation.type == Conversation.Type.ONE_ON_ONE) {
+                            conversationRepository.getConversationMembers(conversation.id).flatMap {
+                                conversationRepository.fetchMlsOneToOneConversation(it.first())
+                            }
+                        } else {
```

Comment:
During testing I realised that backend doesn't allow fetching unestablished 1-1s using the regular `GET /conversations/domain/uuid` API, I suppose they aren't really created until they are established.